src/index.html

```diff
@@ -15,8 +15,101 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo_link"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO_logo"
+          class="moyo_link"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
```

Comment:
Fix it everywhere if you have only 2 attributes
```suggestion
            <a href="#" class="nav_link">
```